src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -1765,7 +1770,7 @@ public FormValidation validate(LDAPSecurityRealm realm, String user, String pass
                                 .LDAPSecurityRealm_UserLookupManagerDnPermissions(),
                         jenkins.security.plugins.ldap.Messages.LDAPSecurityRealm_UserLookupSettingsCorrect());
                 // we do not flag these errors as could be probing user accounts
-            } catch (LdapDataAccessException e) {
+            } catch (AuthenticationException e) {
```

Comment:
The old `LdapDataAccessException` meant that there was problems communicating with the ldap server, or your query was wrong or similar. Is the new `AuthenticationException` equivalent? By the name it sounds to me like it's a different exception.

Reply:
There is no more `org.springframework.dao.DataAccessException` or its subtypes, so in the core PR as well as here I have converted these to the more generic `AuthenticationException`. Whether this is the most appropriate type or not, I do not know, and I am not sure we really care.